### PR TITLE
7513 fix scalability tests

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

#7513 

medic/cht-core#[number]

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/scalability/run_suite.sh [hunk 1]
```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+# Fail job when script errors
+set -e
```
Comment (dianabarsan) [inline tests/scalability/run_suite.sh:3]: This doesn't always work, at least not in the way that it's described. 
For example, it does not apply to commands that are in a && or ||  list ([documentation](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html))

To test this, you can run: 
```
#!/usr/bin/env bash

set -e

cd ./nonexistent &&
echo "one"
echo "two"
echo "three"
```

You'll see that `cd ./nonexistent` fails, so `echo "one"` is not executed, but the script does **not** exit and you get both `echo "two"` and `echo "three"` commands outputs. 

I can count at least 3 instances of "&&" in this file in just this small sample view visible in GitHub review UI. 
This risks to become super confusing when someone tries to debug. 

Please remove all "&&" command lists and keep the `set -e` OR add exit code checks for all commands that can fail (which some say is the only correct way to do it).

**Review by dianabarsan — CHANGES_REQUESTED** [review]
Nice work! I left one comment inline.

File: tests/scalability/run_suite.sh (hunk 1 above)
Comment (ngaruko) [reply tests/scalability/run_suite.sh:3]: removed those `&&`

**Review by dianabarsan — APPROVED** [review]
Nice! Fingers crossed :)

**Pull request merged**